module/chunks/chunkVerifier.go

```diff
@@ -198,7 +202,23 @@ func (fcv *ChunkVerifier) verifyTransactionsInContext(context fvm.Context, chunk
 		return nil, nil, fmt.Errorf("cannot calculate events collection hash: %w", err)
 	}
 	if chunk.EventCollection != eventsHash {
-		return nil, chmodels.NewCFInvalidEventsCollection(chunk.EventCollection, eventsHash, chIndex, execResID), nil
+
+		for i, event := range events {
+
+			fcv.logger.Debug().Int("list_index", i).
```

Comment:
```suggestion
			fcv.logger.Warn().Int("list_index", i).
```

Reply:
I think we could keep this always on. If it never happens again, then no additional log. If it did happen again in the future, we have the log there, otherwise we had to turn on the flag and retry again.

Reply:
Makes sense, updated